proj4/src/main/scala/geotrellis/proj4/io/wkt/Tree.scala

```diff
@@ -0,0 +1,43 @@
+package geotrellis.proj4.io.wkt
+
+case class Authority(name: String, code: String) extends Leaf
+
+case class ToWgs84(values:List[Double]) extends Leaf
+
+case class Parameter(name: String, value: Number) extends Leaf
+
+case class Axis(name: String, direction: String) extends Leaf
+
+case class TwinAxis(first: Axis, second: Axis) extends Leaf
+
+case class Spheroid(name: String, semiMajorAxis: AnyVal, flatteningInverse: Double, authority: Option[Authority]) extends Tree
+
+case class Datum(name: String, spheroid: Spheroid, towsg: Option[ToWgs84], authority: Option[Authority]) extends Tree
+
+case class Unit(name: String, conversionFact: Number, authority: Option[Authority]) extends Tree
```

Comment:
`Unit` is a type in scala `Predef` package which is signifies no return. We should avoid using it here even through its awkward. I would suggest renaming it to `UnitField`.